### PR TITLE
Create seeds

### PR DESCRIPTION
**Pull request by marioh5700**
Changes:
---
 - Add seed data in seed/data/
 - Add executable file to seed db
 - Add makefile command to seed data
---

Minor
---
 - Change username pw for pgadmin
 - Added instructions in readme
---

Fixes
---
 - Fix mistakes in migrations
 - Remove useless text in readme
---

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -20,8 +20,8 @@ services:
     image: dpage/pgadmin4:5.3
     restart: always
     environment:
-      PGADMIN_DEFAULT_EMAIL: admin@email.com
-      PGADMIN_DEFAULT_PASSWORD: secret
+      PGADMIN_DEFAULT_EMAIL: postgres@email.com
+      PGADMIN_DEFAULT_PASSWORD: championsclub123
```
Comment (chunharrison) [inline docker-compose.yml:24]: it's crucial to have these as environment variables

Comment (marioh5700) [reply docker-compose.yml:24]: shut the fuck up these are localhost 🐱‍🐉

Comment (marioh5700) [reply docker-compose.yml:24]: you can comment this on your postgres setup if you want

Comment (chunharrison) [reply docker-compose.yml:24]: just make sure to implement it for your next changes since its not up in prod.

**Review by chunharrison — APPROVED** [review]

**Pull request merged**